api/envoy/extensions/filters/network/ext_authz/v3/ext_authz.proto

```diff
@@ -51,4 +51,7 @@ message ExtAuthz {
   // version of Check{Request,Response} used on the wire.
   config.core.v3.ApiVersion transport_api_version = 5
       [(validate.rules).enum = {defined_only: true}];
+
+  // Flag to specify whether dynamic metadata should be emitted. Defaults to false.
```

Comment:
Do we really need this flag? What is the use case that you want configure this to false when an ext_authz server is returning metadata?

Reply:
When there is no subsequent filter to actually consume the metadata (i.e. when `ext_authz` is the sole gatekeeper).

Reply:
Then shouldn't the ext_authz server just be configured to not return metadata?

Reply:
That’s a fair observation. I can remove this field and update the docs then.

Reply:
Updated.